integration_tests/const_03.py

```diff
@@ -0,0 +1,12 @@
+from ltypes import i64, f64, Const
+
+CONST_1: Const[f64] = 32.0
+CONST_2: Const[f64] = CONST_1 * 2.0
+CONST_3: Const[i64] = CONST_1 + CONST_2
```

Comment:
Let's flip the numbers, so that they are ordered (by name?) and ensure the dependencies actually work:
```suggestion
CONST_3: Const[f64] = 32.0
CONST_2: Const[f64] = CONST_3 * 2.0
CONST_1: Const[i64] = CONST_3 + CONST_2
```

Reply:
Variables don't have dependencies right now. But when I will add dependencies to variables I will use this as a test.